### DISCUSSION TITLE
Avoid generating report based on first run

### DISCUSSION THREAD

**Discussion by ralcar** [GoogleChrome/lighthouse-ci#594] [Q&A]
Im running lchi on my NextJS project and everything is working fine.
The only issue I have is that my report is generated based on the first run, which every time for me will get a "incorrect" score due to the SSR compiling the pages. 
In the report i get a very high "Reduce initial server response time" warning on performance for the first run, which i will not get on the 2nd or 3rd run.

![image](https://user-images.githubusercontent.com/10072001/116203600-270d6e00-a73c-11eb-9565-c463fd7d20c6.png)

I looked through the documentation, but cannot find any way to generate the report based on either the 2nd or 3rd run, or even on the "best performance" run (or perhaps the middle one?). 

Is there any way to achieve this, or get around my problem?

**Comment by patrickhulce** [comment]
> The only issue I have is that my report is generated based on the first run

Which report are you referring to? If it's the uploaded report to temporary storage, LHCI always selects the *median* run of your set to upload for a given URL. If you're still seeing an outlier report selected as the median, you can increase the number of runs.

Faster alternative to this, warm up your server before asking Lighthouse CI to collect results. That way 100% of the runs that Lighthouse sees are what you're trying to measure.
